### PR TITLE
fix(data): Gathering map files into a single folder for ease of use and reference.

### PR DESCRIPTION
**Pull request by Zitchas**
The map file split got shoved through in another PR, so now this one is back to folderization:

**Summary:**
This PR creates a `map` folder and moves the following into it:
- `atlas.txt` (formerly called `map.txt`)
- `planets.txt` (contains all the planets extracted from the previous `map.txt`)
- `stars.txt`
- `landing messages.txt`
- `harvesting.txt`
- `hazards.txt`
- `commodities.txt`

The point is to minimize confusion and make the data structure easier to understand by collecting all the files that establish the galaxy map in one location.

Note: No actual content was changed in this PR. Everything should behave exactly the same in-game; this simply make the location of the data easier to understand.

### CODE REVIEW

**Comment by Zitchas** [comment]
As per Amazinite's suggestion, including the `landing messages.txt` in the map folder.

I considered including `harvesting.txt` and `commodities.txt` but those seemed more economic than galaxy-layout related. Amazinite also pointed out that those are used more by missions than they are by mapping, although there is some cross-over.

**Comment by Zitchas** [comment]
*sigh* What broke? All the tests look good. Last time I tried rebasing, things broke a *lot*

**Comment by tehhowch** [comment]
> *sigh* What broke? All the tests look good. Last time I tried rebasing, things broke a *lot*

The base branch is 0.9.13, and you merged in a different branch (`master`), which means your PR to 0.9.13 now includes your intended commits and any commits in master that aren't in 0.9.13 yet.

**Comment by Zitchas** [comment]
Aren't all the commits in 0.9.13 going to be merged into master? Shouldn't that all work out just fine?

Or should I just re-do this but based on 0.9.13? (I could try rebasing, but I haven't had any success with that yet.

**Comment by tehhowch** [comment]
> Aren't all the commits in 0.9.13 going to be merged into master? Shouldn't that all work out just fine?
> Or should I just re-do this but based on 0.9.13? (I could try rebasing, but I haven't had any success with that yet.

Rather than rebase, or recreate (your branch was already based on 0.9.13) without the merge, the simplest solution is to not merge this PR before 0.9.13 is merged to master and we've retargeted this branch to master.

**Comment by Zitchas** [comment]
OK.

**Comment by MasterOfGrey** [comment]
Let's do the thing, cause it logical.

**Review by Amazinite — DISMISSED** [review]
> Just create a new pleiades.txt file with the relevant data and leave it in the main data folder. Decided against it because that adds to the clutter of the main folder, and this seems a good time to follow in the trend of organizing relevant content together.

I'd prefer this change alone (using the name "pug galaxy.txt" as that is more descriptive of what the file actually contains), as with the addition of species subfolders there isn't actually any significant clutter in the main data folder to worry about. It is notable that I only made subfolders for species, not anything else. Creating a map subfolder (that admittedly doesn't have much in it and I don't see why it would gain any content in the future, while all the species subfolders can be expanded with say new campaign files) would break the pattern of only having subfolders for species.

**Comment by Zitchas** [comment]
The reason for giving them actual names instead of just generic "map" and "pug map" is to help solidify them as real places and not just things that exist for the player's convenience. "pug map" does, indeed, spell out exactly what the file has in it; but I'd like to think that - eventually - it will have more importance than strictly the name of the current primary resident. Giving both map files the name of the galaxy they represent helps establish them as actual entities in their own right and not just objects that are dependant upon the player.

**Comment by Amazinite** [comment]
Okay, definitely no on the latest commit, because that borks the map editor.

**Comment by Zitchas** [comment]
I will swap that back for now, but that's the direction I'd like to see it go - especially once Darcy gets their map viewer up to being a map editor (and supporting multiple files). Just like to note for the record that the Map viewer is already borked as far as "showing all vanilla content" goes.

**Comment by Zitchas** [comment]
There you go. Reverted.

**Comment by MCOfficer** [comment]
> I will swap that back for now, but that's the direction I'd like to see it go - especially once Darcy gets their map viewer up to being a map editor (and supporting multiple files). Just like to note for the record that the Map viewer is already borked as far as "showing all vanilla content" goes.

It can show parts of it, but this change would split planets from their respective systems. I don't even know if the map editor would load that.

**Comment by Amazinite** [comment]
I've gone ahead and simply moved the pug systems to their own file in `data/`, where the current map file is. https://github.com/endless-sky/endless-sky/commit/4af314b5529e4b68605b2a1cfc4dc661a1894991
We need to work with the current limitations of the map editor that we have, and the map editor currently can't handle subfolders. (It can't find any of the images because it's expecting to only have to leave the current folder to find the images folder.) If the map editor is updated to handle this in the future then we can perhaps have a discussion on changing things, but even then I wouldn't be interested in a map subfolder. There is no need to create subfolders for any and every possible subject. Keeping subfolders limited to only factions makes sense to me, especially seeing as there is no issue with clutter now that the faction subfolders exist.

**Comment by Zitchas** [comment]
I will accept your logic in regards to the necessity of the current editor, but I respectfully disagree in regards to the value in having all the map related elements isolated into their own folder: The system, planet, and related files are just as important to the game as any one of the factions, and it would greatly facilitate dealing with it if it was treated as such. 

I will be keeping this branch active and updated so that it can be re-opened whenever someone gets around to improving (or replacing) the current map editor.

**Comment by Zitchas** [comment]
#5163 is the first of the listed alternatives, and has been implemented as a separate PR due to the limitations of the map editor over-ruling the benefits of organizing the data into separate files for general human use.

**Comment by Zitchas** [comment]
Just a note: The branch that this PR comes from still exists, and is up to date as of 2022-Aug-20. So if anyone ever gets the map-editor to handle sub-folders and/or multiple files; it can easily be re-opened for merging.

**Comment by Zitchas** [comment]
I've also added commodities.txt, harvesting.txt, and hazards.txt into the map folder, as these are quite clearly the domain of the map and systems. I previously considered moving them over and decided against them they were more part of economics than mapping; but since then I've reconsidered. Economics is a key part of mapping, after all. And hazards didn't exist at the time.

**Comment by Zitchas** [comment]
Regarding the map file itself: All the map and planet info is currently in one file, because that's all the official map editor understands. I'm leaving it this way for now so that this PR can be merged quickly and easily if someone manages to implement tolerance for subfolders but not multiple files.

If, on the other hand, someone manages to implement multiple files too, then I'm of the opinion that we should split that file in two, and I can do so in here quite easily.

**Comment by Zitchas** [comment]
In view of https://github.com/endless-sky/endless-sky/pull/7986

Given that we are now discussing splitting the map file, which will break the current map editor and that was the one objection to this PR resulting in it being closed, then it is now appropriate to re-open this.

**Comment by Zitchas** [comment]
I pretty much just have to undo the merging of the planets and systems back into one file (as I had already separated them out earlier), and this is good to go.

**Comment by Zitchas** [comment]
Done. 

Question: Should it just be "planets", or specifically "milky way planets", as the pleides planet is stored with that galaxy. I think just planets is simpler, since it is the default galaxy, and where 99% of all our planetary efforts are going to be.

**Review by Hurleveur — APPROVED** [review]
I think that there currently is way too much in the main folder, this would help seeing clearly in it.
Also since you included the changes from Amazinite's PR this looks really good

**Comment by Zitchas** [comment]
Someone during the stream suggested putting an `_` in front of "maps.txt" so that it'd always be the first folder. What do people think of that? What does @Amazinite think of that? It would solve the concern that he had about maps being in the middle of a bunch of faction names. 

I personally see the folders in `data` as "things to gather together collections of files that go together", and not "things to gather faction stuff together." It just so happens that factions are the first things to have gotten so big that they need a folder. 

I'm good with it either way. If it solves the mental/perception issue of not liking it in the midst of the factions, I can make that change happily enough.

---

As for the number of things required to "deserve" a folder, let me just point out that we've already well established a precedent of small numbers of things in folders:
- iije: 1 file
- rulei: 3 files
- whispering void: 2 files
- pug: 1 file (although this could be split up into 4 files if we really wanted to, just they'd be very short files.)
- drak: 4, one of which is the indigenous that may or may not actually really belong with the Drak.

In contrast, this map folder has 7 files. Of the 15 existing folders in data, only 7 of them have 7 or more files within them. So that puts the `map` folder on the larger side in terms of numbers of files.

---

The main obstacle, however, is @quyykk 's plug-in editor, and specifically the question of whether or not splitting up the map file and putting it into a folder like this would cause it any problems. Attached to that is the question of whether or not we make it the official map editor and depreciate the old map editor.

**Comment by MasterOfGrey** [comment]
> Someone during the stream suggested putting an `_` in front of "maps.txt" so that it'd always be the first folder. What do people think of that? 

That's the file name notation to allow plugins to edit pieces of an existing file. I don't think that's either safe or appropriate for a primary vanilla file.

**Comment by Zitchas** [comment]
Sorry, I mistyped. The `_` was in front of the maps *folder*, not the text file.

Does that still apply? Where is that functionality for the underscore documented? I've never heard of that before.

**Comment by MasterOfGrey** [comment]
Ah, yeah I don't know about the folder. That's beyond me.
I just know I picked up the functionality of the file stuff waaaay back when I was editing maps in plugin form for testing using I think Quyyk's editor.

**Comment by Zitchas** [comment]
Interesting. Good to know. Although I kind of thought that's what our plug-in architecture did by default. As in, anything in the plug-in folder basically just got appended onto the data files, effectively.

**Comment by Zitchas** [comment]
Doing a quick test of @quyykk 's editor:
- Downloaded the latest version (says last updated 2022-12-22)
- Copied the exe and libraries into the Endless Sky folder (didn't overwrite anything already present. Not sure if there's version differences there)
- This is using my git development folder, which is currently set to this PR (5041)

Checks:
- [X] Plug-in Editor runs.
- [X] Galaxy map opens
- [X] All systems appear to be present
- [X] Random spot check shows systems have planets.
- [X] Random spot check shows systems have commodity pricing.
- [X] Look at some random planets to see if they have descriptions, outfitters, and shipyards
- [X] Try modifying Earth's data (add to the description and add an outfitter)
- [X] Try saving the data (triggers dialog box asking for a name for a new plug-in)
- [X] Verify that the plug-in actually saved.

All this ran successfully. Anyone else have any other suggested things to check that are relevant to the map related files being collected into one folder and the map file itself split in two?

@Amazinite do you have suggestions about map-related stuff you'd like to see tested?
As far as that goes, would having the underscore in front of the map folder be beneficial to you?

**Comment by warp-core** [comment]
> As for the number of things required to "deserve" a folder, let me just point out that we've already well established a precedent of small numbers of things in folders:

I'm not sure the number of files each folder ends up with is the decider for whether or not those files should get their own folder.

The stuff that is sorted into subfolders is all largely self contained content, not a fundamental thing that is used throughout the game. Stuff in other folders may make use of it, the map makes use of some of it, but that's not the defining characteristic.

The map, on the other hand, is fairly universal. Wherever you are, you *are* making use of it.

I don't think the main data folder is so cluttered that these files need to be moved into a folder at this time, and even if I did, I don't agree that all of these files belong here. I don't think commodities has a particular strong relation to the map, most of it is actually devoted to content used by cargo missions and jobs.

If we are to move the files not currently in a subfolder into subfolders, why not also move some of the others? Tooltips, ratings, interfaces, and help could all go into a UI folder (arguably also where landing messages should go). Dialog phrases and conversations could go to a "generic text" folder (another place landing messages would fit, and I can see some potential reasons why commodities would also fit).

**Comment by Zitchas** [comment]
Well, we can group those ones too, if you want. If Amazinite gives some kind of indication that he wants those grouped too (Maybe his thumbsup on  your post counts?), I can include those.

As far as commodities goes, I associate it with the map because I see commodities on the map every day. hundreds of times per session. And every instance of those commodities being used in jobs that I see on a regular basis is *also* on the map screen. True, they can be seen elsewhere, but the vast majority is in jobs, which are always accepted on the map. It's perhaps a prime example of things that are extremely map-centric that just happens to be referred to by stuff elsewhere sometimes.

**Comment by warp-core** [comment]
> Well, we can group those ones too, if you want. If Amazinite gives some kind of indication that he wants those grouped too (Maybe his thumbsup on  your post counts?), I can include those.

I suspect his thumbs up is supportive of my point that these files don't really need to be grouped into a subfolder.

> As far as commodities goes, I associate it with the map because I see commodities on the map every day. hundreds of times per session. And every instance of those commodities being used in jobs that I see on a regular basis is *also* on the map screen. True, they can be seen elsewhere, but the vast majority is in jobs, which are always accepted on the map. It's perhaps a prime example of things that are extremely map-centric that just happens to be referred to by stuff elsewhere sometimes.

The map file also contains many references to fleets, and the vast majority of ships we see in game are spawned based on the system definitions in the map file. But, that's not a reason to separate fleet definitions from their current groupings.

Because, that's what this does, it separates these files from everything else. The contents of the other folders are separate things that happen to have overlap. The map overlaps with everything.

**Comment by Zitchas** [comment]
I'm sorry, but while I understand what you are saying, I don't see it that way. Just because things happened to be referenced by a few things doesn't mean it has to be stored at the room level. Especially once the map file is split. I can't speak for everyone, but evidently at least a few people agree that it makes sense to have these gathered together in one place. If I'm dealing with the systems file, I'm almost guaranteed going to be dealing with the planets file. Hazards, stars, and harvesting quiet likely too, if only to reference. Being able to tell what landing message goes with what stellar object just makes sense to me.

These files belong together just as much as void sprites belong with Drak stuff, for instance. 

As for the thumbsup/down, I agree, I was just pointing out that using thumbsup/down can be misconstrued and it'd be better if he just talked. His statements have sounded like preferences to me. As in he didn't have strong feelings on the subject. he didn't like it, but didn't strongly care either way about it. 
So I've been making changes as per his requests: 
- He wanted the Pug stuff merged back into the systems and planets file, so I did it. 
- He wanted the files renamed to not be galaxy specific, I did it. 
- He wanted the map folder to not be in the midst of the faction names, so I'm ready to implement a solution if that's acceptable to him. 
- He wanted testing, so I got Quyyk's plug-in editor installed and working and did some testing with this, as documented above. 
- I've asked for feedback as to whether all this is acceptable to him, and I'm getting radio silence.
- If he can point out anything else that needs testing, I will happily do it.

I feel like he's provided review feedback that I can work on, and I've done what was asked, and now I'm sitting here wondering if I'm just being run around a loop to waste time.

**Comment by Amazinite** [comment]
I will note that splitting of the map file and creating a map folder are two separate things. Of the four "He wanted" points, three of them are related to the splitting of the map file, and the fourth (map folder amongst faction folders) is part of the reason I don't want a map folder.

**Comment by Zitchas** [comment]
That's still three things you asked for, that I have given you here. Given that you haven't specified anything else, I assume that those and the testing are sufficient to satisfy you in that regards?

And you didn't answer the question as to whether using underscore (or any other character / name combination) to ensure the folder is at the start (or end) of the list would mitigate that negative. As best I can tell, this is strictly an aesthetic thing on your part, versus a strongly beneficial functional/utilization benefit on our part. 

Just to be clear, *you* don't want the folder. Apparently warpcore doesn't either. Petervdmeer, Masterofgrey, Hurleveur, and myself all consider it to be a benefit. 

Also "part of the reason?" What other reasons are there? Is there something we can work on here? How can this be improved?

**Comment by Hecter94** [comment]
Given that moving the map into a subfolder appears to be a controversial change, whereas simply splitting the map file is something that most seem to agree with, and is something we could do right now, and would resolve the issue with GitHub being unable to display it. 

I would suggest we move ahead with splitting the file asap to resolve the GitHub issue, and can then move the split files into their own subfolder in a follow-up PR if we do decide that would be a beneficial thing to do.

**Comment by Zitchas** [comment]
I think you and I have very different definitions of what constitutes a controversy. There's no controversy here, merely a difference of opinions and development styles. I prefer to have things at least loosely grouped and think that seven things that go together is more than sufficient to beneficial to my development practices; and that likewise having those seven lumped in with 13 other basically unrelated files is needlessly mixing things and has a negative impact on my development process. Amazinite disagrees.

Amazinite mentioned in the stream working on java projects and developing an antipathy towards having stuff nested or buried in long cascades of folders. Which I can very much appreciate from my own professional experience. Dealing with projects that have >6 levels of folders inside them, especially when they merely have one or two files each, is very cumbersome. However, there is a very significant difference between using a single depth of folders to group related things together and implementing a complex system where everything is nested potential 6+ folders deep. These are basically extreme opposites, and the fact that he brought it up raises concerns on my part that his antipathy towards java's (or his company's) mandated folder structure is jeopardizing our ability to have even a minimal level of sorting.

Perhaps this is fear is emphasized by the fact that we have demonstrated falling into the folder structures promulgated by some project systems in the creation of two folders to hold but a single file each already. Sure, they will *probably* have more files someday, but then again, maybe not.

**Comment by Hecter94** [comment]
> I think you and I have very different definitions of what constitutes a controversy. There's no controversy here, merely a difference of opinions

**con·tro·ver·sial**
_adjective_
giving rise or likely to give rise to public disagreement.



So to clarify, are you against the idea of just splitting the map file now to get the file viewable in GitHub again and then discussing the merits and demerits of putting those split folders into their own subfolder in a followup PR? That wasn't clear from your message.

**Comment by Zitchas** [comment]
OK, to clarify: I consider the label "Controversy" to apply to stuff that is a little more substantial. For example, whether or not we should include explicit references to how the Alphas reproduce; or discussing how much of a presence religion should have in the game. Those are controversial topics that are very sensitive to some people. These are things that if we choose poorly can cause people serious problems, and could even cause the project as a whole to get banned in certain countries, organizations, and households.

Mere differing opinions as to what constitutes "cluttered" is not a controversy, it's just a disagreement. 

Let's be clear: The map file has been a major pain to deal with for *years* now. In fact, in this very PR, I initially proposed splitting it two years ago. Ironically, it was vetoed by Amazinite, who also suggested including the landing messages in the folder, too. Granted, he had good reason to, as we were still kind of dependant on the map editor back then. Although many of us (including myself) only used it to get rough coordinates and then just transposed those by hand into files anyway; so it wasn't as if only having it read the systems with the planet info would have seriously changed anything for many of us. But making the split then so that we could work effectively with the map file and using that breaking change to push actual work on either the map-editor or a new editor could have been very beneficial to the project.

So yeah, while I am clearly in favor of splitting the map file, I am not in favor of rushing to do it until we have the clutter sorted a bit. Otherwise, this will just be shoved to the backburner, and then in a few months it'll be closed again, and it'll be another couple years until we actually reach a breaking point and someone (probably Amazinite) decides to do something about it. It would be nice if, this time, we actually take the time to work it out and prevent the problem from ever reaching that point instead of just pushing the buck into the future hoping that it never comes.

tldr: Splitting the map file has waited two years, it can wait a few more days.

**Comment by warp-core** [comment]
> tldr: Splitting the map file has waited two years, it can wait a few more days.

For those two years it was still possible to view it in GitHub in a browser.

**Comment by Hecter94** [comment]
> OK, to clarify: I consider the label "Controversy" to apply to stuff that is a little more substantial.

Okay; I typically use the official definitions of words, and not my own personal definition, so that explains the confusion.

> So yeah, while I am clearly in favor of splitting the map file, I am not in favor of rushing to do it until we have the clutter sorted a bit.

Presumably this "clutter" has been around for a while; I see no reason why its supposed existence should be a blocking factor for an entirely separate, and important, fix.

Is this "clutter" that you see in the folder a major enough issue to block a fix for an actual issue with GitHub that's preventing us from viewing the diffs on a very important file?
Is there a reason why fixing the map file to allow us to view it in GitHub again can wait, but fixing this "clutter" can not?

**Comment by Zitchas** [comment]
Instead of arguing over whether or not we could have seen this coming and taken steps to deal with it while it was merely annoying instead of a crisis; why don't we discuss how we can take steps to proactively make dealing with the rest of the contents in the data folder better? Better to resolve two problems in a couple days than to solve one now and argue incessantly about the other for the next year.

**Comment by Zireael07** [comment]
I agree with @Hecter94 in that your worry about 'clutter' seems to be blocking a fairly important fix

**Comment by 1010todd** [comment]
As someone who constantly go into ES data folder almost everyday at this point, the data folder do feel a little cluttered to me while doesn't really take more than few second to find things but putting a few map stuffs into its own folder would help. imo map folder(if it's going to happen) should be just the map: systems, planets, stars, hazard, and harvesting definitely sounds like it should belong in there. Commodities is also used for other stuffs (mainly missions) so probably should stay outside, also not all of things in commodities.txt are used in the map. Other than that it might be a good idea to future-proof in case we got more things in data folder later and just put map things in map folder now instead of later but it's not that urgent right now I guess.

**Comment by Zitchas** [comment]
Not really. Potential paths forward:
- Amazinite just hits merge on this. Both problems solved.
- People weigh the impacts. Currently it is "Better organization, and several people think it is a net gain to the project and/or their own personal productivity here" vs "A couple people don't think it is a problem or just don't like folders."
- People suggest changes to this to help it move forward.

I've been pretty clear about willingness to make changes, so long as the results are that the core files relevant to the galaxy are isolated. I've even put together another PR that could either complement this one or be an alternative to it. So I've got two potential paths forward. 

Oh, as an alternative to the `map` folder name: `atlas`. Not sold on the name myself, but it is an alternative that would be a bit clearer that this folder is about the structure of the galaxy, not just specifically the map panel.

**Comment by Hecter94** [comment]
> * Amazinite just hits merge on this. Both problems solved.

Not a viable option due to at least part of this PR being a controversial change.

> * People weigh the impacts. Currently it is "Better organization, and several people think it is a net gain to the project and/or their own personal productivity here" vs "A couple people don't think it is a problem or just don't like folders."
> * People suggest changes to this to help it move forward.

Should be done in a followup PR to avoid blocking an otherwise important fix.

It seems like the correct path forward here is just to merge #7986 to resolve the issue asap, allowing this PR to focus specifically on the discussion regarding moving the map files into their own folder, without that discussion being needlessly rushed due to it being bundled with a largely unrelated fix or that fix being delayed due to discussion about a largely unrelated change.

**Comment by Zitchas** [comment]
Needlessly rushed? lol. It's been around for longer than you have. It just so happens that *last* time this discussion was brought up, Amazinite went "No, we don't need to split the map file. And since we don't need that, and I'm not a fan of the folders, I'll just close this."

**Comment by warp-core** [comment]
> Amazinite went "No, we don't need to split the map file. And since we don't need that, and I'm not a fan of the folders, I'll just close this."

I think the important part of what he said was actually:
> We need to work with the current limitations of the map editor that we have, and the map editor currently can't handle subfolders.

He did say other things, but really, given the issue with the map editor, that's a pretty objective negative to merging this at the time.

I agree with Hecter, split the planets out of the map file with the other PR, and consider reorganising the files into subfolders as a separate issue to that.

**Comment by Zitchas** [comment]
As far as that goes, it's one thing to redo the content of an abandoned PR whose author isn't around any more and re-PR it yourself. It's quite another to forcibly close someone's PR and then years later re-purpose a large chunk of what they did into your own PR. The courteous thing to do would be to re-open the existing PR that tackled that specific issue and discuss it, maybe ask for it to have the parts one didn't like removed while offering to have a discussion about alternatives to keeping the folder a bit tidier. 

If he'd done that... For that matter, if he *did* do that, I'd be happy to make those changes, watch this get merged with just the split and no folders, and then move on to discussing this in an issue and working out a solution that gets PR'd in a a couple weeks. I'm good with that.

So far, though, that's not what's happening.

**Comment by warp-core** [comment]
Well, you can see exactly what Derpy wants: it's in his own PR.
Is that unclear to you? Do you need to be explicitly told?
I'm sure Derpy can't remember every PR, nor every PR that he's commented on.
You have not, until now, even suggested that you would be willing to make the changes to this PR to match what is being looked for, and indeed have asserted that this is the better change.

When you asked him if he was going to mark that as a duplicate, he said that he was not because his PR does something different to yours, and that continues to be the case.
Is it still the case that this PR is not the same as that PR.
The fact that almost all the line changes in this PR are essentially the same as those in that PR is really just a product of the way GitHub presents changes. Splitting a text file down the middle would actually only involve inserting the relevant metadata and headers, etc at the relevant point and moving everything else down a little bit. In that regard, the moving of other files is just as significant.

**Comment by warp-core** [comment]
> Are we not explicitly told to check to make sure we're not duplicating stuff? We have a checkbox and a section for it in the issue template. Maybe we need a "Did you check for other PRs that cover this topic" on the PR template too?

Well, again, that PR is not actually the same as this PR, and you have given no indication that you are willing to change this PR to match that PR. You could do it now and I'm sure Derpy would be willing to close his and merge this instead.

You keep asking for suggestions, but have ignored the one that was made clear by Derpy's response to your question on stream.

**Comment by Zitchas** [comment]
As far as that goes, I actively updated it in August:

> Just a note: The branch that this PR comes from still exists, and is up to date as of 2022-Aug-20. So if anyone ever gets the map-editor to handle sub-folders and/or multiple files; it can easily be re-opened for merging.

And then after he opened his:

> In view of https://github.com/endless-sky/endless-sky/pull/7986
> 
> Given that we are now discussing splitting the map file, which will break the current map editor and that was the one objection to this PR resulting in it being closed, then it is now appropriate to re-open this.

To be blunt, Amazinite is the one with who raised the topic, he's the one who originally closed this. I'd like to have some actual conversation with Amazinite instead of trying to guess whether his statements mean he's "not a fan of the folders, but is open to it." or if the fact that there is a folder is a hard never-going-to-happen and I've been wasting a lot of time and effort over the past few days.

It's not my job to jump back and forth and make guesses as to what Amazinite wants. Frankly, I'm not particularly fond of having you serve as the Aaron to Derpy's Moses, either. Is he just looking for a good reason for having a folder? Is having it prefixed so it stands out good? I don't know! I asked him directly, and all I got was a general "the fact that the map folder would be in the midst of the faction folders is part of why I don't want it." Which doesn't answer the question, and leaves hanging that "what is the other part?" All I know is that he has expressed a general aesthetic preference for not having folders. If he came in here and said "this gets in only without folders," then, well, I know what I have to do to get the first half of the changes I wanted to do. But that's not what I'm hearing. Everything I hear is a vague "Not a fan of folders, but..." that rather implies that there's potentially something that I could do to improve the situation, or an alternate route to getting the whole thing settled. I can't think of it, though.

But sure, I'll do it. I'll remove the folderization. Will you commit to being the first to approve it, and then think up some ideas for solutions you can post on an issue discussing the cluttered nature of our data folder?

**Comment by Zitchas** [comment]
*sigh* I prefer systems, since it is explicitly stating the primary contents of the file. I seem to recall someone suggested changing maps *to* systems a while back, which is part of the reasons it is what it is. 

If Amazinite or a number of other people agree with that, then sure, I can change that.

**Comment by Hecter94** [comment]
> I would like to see systems.txt retain the "map" name, as without context I feel like a file named "systems" in a game's data files could be confusing.

My initial first thought would probably be that it's referring to the outfit category, not the map.

**Comment by 1010todd** [comment]
I agree `map` would be clearer than `systems`, people who regular look at data files already know anyway and for newcomers it'll still be clear that it's at least where system position in the galaxy map or the in system "map" is defined. (If not searching through 5 more files shouldn't discourage them from doing what they want else there's nothing we can do anyway.)

**Comment by Amazinite** [comment]
Closing in favor of #7986. The other PR has a more preferred naming scheme for the files, as well as touches up on some formatting issues in the existing system definitions. As noted in that PR, other organization changes beyond just splitting the map are a separate discussion.

**Comment by Zitchas** [comment]
Well, since the names got plowed through just as people were starting to discuss the names, this is going back to folderization.

I agree that systems has the potential for confusion. What do you think about "atlas" for the name of the file that is currently called systems?

**Comment by Saugia** [comment]
We're moving on. There's no need for this petty argument anymore. We merged Derpy's, it's one PR, the world isn't ending, nobody hates anyone, the names *were* discussed and people prefer the "map" part. Zitchas, you have to stop. This is getting ridiculous.
